Podfile

```diff
@@ -20,7 +20,7 @@ target 'WooCommerce' do
 
   # Use the latest bugfix for coretelephony
   #pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.4-beta.1'
-  pod 'Automattic-Tracks-iOS', '~> 0.4.0'
+  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta'
```

Comment:
Looks like this is for https://github.com/woocommerce/woocommerce-ios/pull/1994/ 🙂

Reply:
Oops. Updated!